src/components/StoreButton.tsx

```diff
@@ -2,12 +2,22 @@ import { JSXInternal } from 'preact/src/jsx'
 import { width } from 'classnames/tailwind'
 import Button from 'components/Button'
 
-const buttonWidth = width('w-64', 'xs:w-80', 'md:w-50')
+const buttonWidth = (small?: boolean) =>
+  width({
+    'md:w-50': !small,
```

Comment:
sort like `w-, xs:, md:`, to make it easier to read the styles

Reply:
It is sorted automatically by alphabet. I can't control it